### PR TITLE
Updating user groups to use new attributes - added new attributes to …

### PR DESCRIPTION
**Pull request by chtyler**
…attributes file
To make it easier to update the default example ODH users and admin group. New attributes are required for ODH (and also will be for downstream OAI). The objective of this PR is to add the new attributes to the ODH attributes and update all occurrences of the old ODH admin/users groups. 

## Description
Added attributes to the ODH attributes file containing the ODH admin and users group. There are occasions where I have also been able to update the OAI admin and users group too. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by chtyler** [comment]
i have tested the site locally and it builds successfully with all the new changes.

**Review by aduquett — COMMENTED** [review]
lgtm

File: modules/cleaning-up-after-deleting-users.adoc [hunk 1]
```diff
@@ -7,20 +7,20 @@
 
 After removing a user's access to {productname-long} or Jupyter, you must also delete their associated configuration files from {openshift-platform}.
 
-It is recommended that you back up the user’s data before removing their configuration files.
+Red Hat recommends that you back up the user’s data before removing their configuration files.
```
Comment (MelissaFlinn) [inline modules/cleaning-up-after-deleting-users.adoc:10]: Is "Red Hat recommends" appropriate for upstream?

**Comment by MelissaFlinn** [comment]
other than my one minor comment above - lgtm, too

File: modules/cleaning-up-after-deleting-users.adoc (hunk 1 above)
Comment (chtyler) [reply modules/cleaning-up-after-deleting-users.adoc:10]: This file doesn't actually appear upstream. It is only included downstream.

Comment (MelissaFlinn) [reply modules/cleaning-up-after-deleting-users.adoc:10]: sorry about that!

**Comment by MelissaFlinn** [comment]
lgtm

**Pull request merged**